2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/.config/wandb/settings
2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/settings
2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_init.py:_log_setup():386] Logging user logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220301_220736-32blq604/logs/debug.log
2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_init.py:_log_setup():387] Logging internal logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220301_220736-32blq604/logs/debug-internal.log
2022-03-01 22:07:36,620 INFO    MainThread:3573 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7fec0075a9a0>
2022-03-01 22:07:36,621 INFO    MainThread:3573 [wandb_init.py:init():420] calling init triggers
2022-03-01 22:07:36,621 INFO    MainThread:3573 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-03-01 22:07:36,621 INFO    MainThread:3573 [wandb_init.py:init():471] starting backend
2022-03-01 22:07:36,621 INFO    MainThread:3573 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-01 22:07:36,635 INFO    MainThread:3573 [backend.py:ensure_launched():219] starting backend process...
2022-03-01 22:07:36,651 INFO    MainThread:3573 [backend.py:ensure_launched():224] started backend process with pid: 3612
2022-03-01 22:07:36,653 INFO    MainThread:3573 [wandb_init.py:init():480] backend started and connected
2022-03-01 22:07:36,672 INFO    MainThread:3573 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-01 22:07:36,673 INFO    MainThread:3573 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-01 22:07:36,673 INFO    MainThread:3573 [wandb_init.py:init():550] updated telemetry
2022-03-01 22:07:36,702 INFO    MainThread:3573 [wandb_init.py:init():581] communicating current version
2022-03-01 22:07:37,561 INFO    MainThread:3573 [wandb_init.py:init():586] got version response 
2022-03-01 22:07:37,561 INFO    MainThread:3573 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-01 22:07:37,783 INFO    MainThread:3573 [wandb_init.py:init():624] starting run threads in backend
2022-03-01 22:07:40,744 INFO    MainThread:3573 [wandb_run.py:_console_start():1827] atexit reg
2022-03-01 22:07:40,744 INFO    MainThread:3573 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-01 22:07:40,745 INFO    MainThread:3573 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-01 22:07:40,746 INFO    MainThread:3573 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-01 22:07:40,746 INFO    MainThread:3573 [wandb_init.py:init():651] run started, returning control to user process
2022-03-01 22:07:40,750 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:40,750 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:40,755 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:40,755 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:40,756 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:40,761 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:40,767 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:40,767 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:40,771 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:40,773 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:40,773 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:40,778 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:43,937 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:43,937 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:43,942 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:43,942 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:43,942 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:43,946 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:43,949 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:43,950 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:43,953 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:07:43,962 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:07:43,963 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:07:43,967 INFO    MainThread:3573 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-01 22:11:50,589 INFO    MainThread:3573 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-01 22:11:50,589 INFO    MainThread:3573 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-01 22:21:48,065 INFO    MainThread:3573 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-01 22:21:48,067 INFO    MainThread:3573 [wandb_run.py:_restore():1769] restore
2022-03-01 22:21:48,989 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-01 22:21:49,095 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 4251
}

2022-03-01 22:21:49,198 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 4251
}

2022-03-01 22:21:49,300 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:49,403 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:49,505 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:49,607 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:49,709 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:49,813 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:49,915 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:50,018 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:50,120 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:50,223 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:50,325 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:50,427 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

2022-03-01 22:21:50,529 INFO    MainThread:3573 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1455
  total_bytes: 4251
}

